client.go

```diff
@@ -898,7 +898,11 @@ func (c *TunaSessionClient) startExits() error {
 	listeners := make([]net.Listener, c.config.NumTunaListeners)
 	var err error
 	for i := 0; i < len(listeners); i++ {
-		listeners[i], err = net.Listen("tcp", "127.0.0.1:")
+		port, err := GetFreePort(0)
```

Comment:
OS will always choose an available port when listening tcp without a port

Reply:
Because we found sometimes TCP port is free, and TCP listener started. but UDP port is not free, and UDP listener can't start. Here `GetFreePort` will check port is free in both TCP and UDP.